src/components/card/Card.jsx

```diff
@@ -0,0 +1,43 @@
+import React, {useEffect, useState} from 'react';
+import "./Card.css"
+import axios from "axios";
+
+function Card({url}) {
+  const [pokemon, setPokemon] = useState([]);
```

Comment:
Ik zou de default waarde undefined maken, want dan heb je een wat duidelijker verschil tussen of het al opgehaald is of niet
`const [pokemon, setPokemon] = useState();`